helm/basemap/templates/cronjobs.yaml

```diff
@@ -98,11 +99,16 @@ spec:
               requests:
                 memory: {{ .Values.resources.db.requests.memory | quote }}
                 cpu: {{ .Values.resources.db.requests.cpu | quote }}
+            # lifecycle:
+            #   preStop:
+            #     exec:
+            #       command: ["sh", "-c", "rm -rf /var/lib/postgresql/data/*"]
```

Comment:
There are commented-out lines in the cronjobs.yaml file. If you want to keep them for reference, provide a clear explanation of why they are commented out. Otherwise, remove these lines:

```suggestion
```

Reply:
Fixed, typo and comments but about will it not work or not I am not sure. I am doing this type of task first time. At least we can check it.

Reply:
About branch name if you mean like as this 15155-basemap-postgres-cleanup then ok.